### PR TITLE
Install akismet

### PR DESCRIPTION
**Pull request by Discordius**

### CODE REVIEW

**Comment by jimrandomh** [comment]
Was based on the posts-page rework. Will wait until that's merged, so that the diff becomes legible, to review.

File: packages/lesswrong/server/akismet.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import LWEvents from '../lib/collections/lwevents/collection'
+import Reports from '../lib/collections/reports/collection.js'
+import { Posts } from '../lib/collections/posts/collection.js'
+import { Comments } from '../lib/collections/comments/collection.js'
+import { newMutation, getSetting, addCallback } from 'meteor/vulcan:core';
+import akismet from 'akismet-api'
+
+async function checkForAkismetSpam({content, author, link}) {
+    const events = await LWEvents.find({userId: author._id, name: 'login'}, {sort: {createdAt: -1}, limit: 1}).fetch()
+    const ip = events && events[0] && events[0].properties && events[0].properties.ip
```
Comment (Raemon) [inline packages/lesswrong/server/akismet.js]: would this be a bit cleaner with .findOne?

File: packages/lesswrong/server/akismet.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            postId: comment.postId,
+            commentId: comment._id,
+            link: Comments.getPageUrl(comment),
+            description: "Akismet reported this as spam"
+          },
+          validate: false,
+        })
+      } else {
+        //eslint-disable-next-line no-console
+        console.log('Comment marked as not spam');
```
Comment (Raemon) [inline packages/lesswrong/server/akismet.js]: Should this include comment id?

File: packages/lesswrong/components/sunshineDashboard/SunshineReportedItem.jsx [hunk 3]
```diff
@@ -27,20 +27,34 @@ class SunshineReportedCommentsItem extends Component {
...
-      })
+    if (confirm(`Are you sure you want to immediately delete this ${report.comment ? "comment" : "post"}?`)) {
+      if (report.comment) {
+        updateComment({
+          documentId: report.comment._id,
+          set: {
+            deleted: true,
+            deletedDate: new Date(),
+            deletedByUserId: currentUser._id,
+            deletedReason: "spam"
```
Comment (Raemon) [inline packages/lesswrong/components/sunshineDashboard/SunshineReportedItem.jsx]: I don't think this should change right now, but I notice that:

a) our workflow should be to basically always mass-purge users who are spamming
b) therefore, the "delete individual post/comment" option should probably be used for things that aren't spam, but instead bad for other reasons, so auto-labeling it "Spam" is a bit weird.

File: packages/lesswrong/server/akismet.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+          userId: currentUser._id,
+          postId: post._id,
+          link: Posts.getPageUrl(post),
+          description: "Akismet reported this as spam"
+        },
+        validate: false,
+      })
+    } else {
+      //eslint-disable-next-line no-console
+      console.log('Post marked as not spam');
```
Comment (Raemon) [inline packages/lesswrong/server/akismet.js]: should this include post._id

**Pull request merged**